### PR TITLE
Issue#38 - Flip Order of Gamma and Beta

### PR DESCRIPTION
**Pull request by rajgane07**
@rsln-s @ZichangHe 

Made changes and tested. Please review and merge

### CODE REVIEW

**Comment by rsln-s** [comment]
@ZichangHe would you mind reviewing? Thank you!

**Comment by ZichangHe** [comment]
I fixed some inconsistent issues in the example notebooks. Otherwise, looks good to me! 

A quick question, how to restore outputs in jupyter notebooks?

**Comment by rsln-s** [comment]
> A quick question, how to restore outputs in jupyter notebooks?

On OmniQ, you can delete hooks: 
```
cd QOKit
rm .git/hooks/*
```

File: examples/QAOA_LABS_optimization.ipynb [hunk 1]
```diff
@@ -2,15 +2,16 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "import numpy as np\n",
     "import qokit\n",
```
Comment (rsln-s) [inline examples/QAOA_LABS_optimization.ipynb:10]: Unused import, please remove!

File: examples/QAOA_LABS_optimization.ipynb [hunk 2]
```diff
@@ -2,15 +2,16 @@
...
     "import scipy.optimize\n",
     "from qokit import parameter_utils\n",
-    "import qokit.fur "
+    "import qokit.fur \n",
```
Comment (rsln-s) [inline examples/QAOA_LABS_optimization.ipynb:13]: Unused import, please remove!

File: examples/advanced/compare_maxcut_parameters.ipynb [hunk 3]
```diff
@@ -48,58 +48,15 @@
...
-      "QAOA Approximation Ration at p = 16\n",
-      "SK: 0.9174003464111532\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# Generate a random regular graph\n",
-    "N = 20\n",
+    "N = 10\n",
```
Comment (rsln-s) [inline examples/advanced/compare_maxcut_parameters.ipynb:59]: Can we put 20 back in? Why switch to 10? 20 should be fast enough with `c` simulator even on CPU.

**Review by rsln-s — CHANGES_REQUESTED** [review]
Thank you @rajgane07 and @ZichangHe! Special thanks to @ZichangHe for fixing the notebooks – it is important we keep them up-to-date.

@rajgane07, may I ask you to fix the comments and rerun the notebooks so that the output is displayed on Github?

**Comment by ZichangHe** [comment]
Fixed the two minor change.

**Review by ZichangHe — APPROVED** [review]

**Review by rsln-s — APPROVED** [review]
LGTM

**Pull request merged**